.coveragerc

```
@@ -0,0 +1,29 @@
+[html]
+show_contexts = true
+skip_covered = false
+
+[paths]
+source =
+  mysite
+
+[report]
+skip_covered = true
+show_missing = true
+exclude_lines =
```

There's a new setting in coveragepy called `exclude_also`. There are default entries that coveragepy sets for `exclude_lines`. If you supply your own list, it replaces those entries, but with `exclude_also`, it extends the default entries with yours. It's considered better now. Also, in the future, it might be nice to integrate https://pypi.org/p/covdefaults which provides many popular settings automatically.
But for now, this small change should suffice:
```suggestion
exclude_also =
```